kolibri/plugins/management/assets/src/vue/modal/index.vue

```diff
@@ -40,27 +44,87 @@
 
 <script>
 
+  const vue = require('vue');
+
   module.exports = {
     props: {
-      btntext: {
+      title: {
         type: String,
-        default: 'Open Modal',
+        required: true,
+      },
+      invisibleTitle: {
+        type: Boolean,
+        default: false,
+      },
+      // Modal options
+      disableClose: {
+        type: Boolean,
+        default: false,
+        required: false,
+      },
+      backgroundClickClose: {
+        type: Boolean,
+        default: true,
+        required: false,
+      },
+      // useed to toggle error message in header
+      error: {
```

Comment:
the name of this prop is a bit confusing - I would have expected the type to be `String`. Maybe instead: `hasError`?